src/olympia/accounts/utils.py

```diff
@@ -113,11 +113,12 @@ def process_sqs_queue(queue_url):
     log = getLogger('accounts.sqs')
     log.info('Processing account events from %s', queue_url)
     try:
+        region = queue_url.split('.')[1]
```

Comment:
`region` really should be validated against available regions to make sure this isn't misconfigured in the future when AWS decides to change it's URL structure.

```python
from boto3 import _get_default_session

available_regions = _get_default_session().get_available_regions('sqs')

if region not in available_regions:
    log.error('SQS misconfigured, expected region, got %s from %s' % (region, queue_url))
```

or something like that. Not too sure how other libraries or apps do it, maybe you can get inspired there (django-storages or so do use boto too)

Reply:
https://github.com/mozilla/addons-server/pull/7974/commits/c04e88ddac6261c945a85550322426116c84f2d7